proto/blockchain.proto

```diff
@@ -34,13 +34,16 @@ message NativeCallItem {
     uint32 id = 1;
     string method = 3;
     bytes payload = 4;
+    string nonce = 5;
```

Comment:
Let's limit it to uint64, that would make it safer to validate

Reply:
That would limit the way I generate a nonce, even my implementation of sender nonce is sha256, it would not fit in uint64. Let's make it bytes too may be?

Reply:
Do you really need more than 64 bits? Maybe you can just have `value & 0xffffffffff`?

The reason for using a number is that I expect that clients would just use a sequence number, i.e., starting from from zero and incrementing it with each request.